cubedash/_stac.py

```diff
@@ -319,11 +319,13 @@ def as_stac_item(dataset: DatasetItem):
         bbox=dataset.bbox,
         geometry=dataset.geom_geojson,
         properties={
+            "collection": dataset.product_name,
             "datetime": utc(dataset.center_time),
             **dict(_build_properties(dataset.odc_dataset.metadata)),
             "odc:product": dataset.product_name,
             "odc:processing_datetime": utc(dataset.creation_time),
             "cubedash:region_code": dataset.region_code,
+            "eo:epsg": ds.crs.epsg
```

Comment:
I can't see an `eo:epsg` in [the stac repo](https://github.com/radiantearth/stac-spec/search?q=eo%3Aepsg&unscoped_q=eo%3Aepsg) -- should it be `proj:epsg` or something? 

(and presumably this will be None on datasets with a non-epsg crs?)

Reply:
I was copying it from the Landsat STAC implementation, see here: https://landsat-stac.s3.amazonaws.com/landsat-8-l1/147/032/2015-01-02/LC81470322015002LGN00.json

But I note that it isn't in the latest `eo` extension, here: https://github.com/radiantearth/stac-spec/tree/master/extensions/eo. So it probably should go.

Reply:
Ah, looks like it was there in the 0.6.0 version of the `eo` extension: https://github.com/radiantearth/stac-spec/tree/v0.6.0/extensions/eo